json-endpoints/jetpack/class.jetpack-json-api-sync-endpoint.php

```diff
@@ -52,7 +52,9 @@ protected function validate_queue( $query ) {
 // GET /sites/%s/sync/status
 class Jetpack_JSON_API_Sync_Status_Endpoint extends Jetpack_JSON_API_Sync_Endpoint {
 	protected function result() {
-		return Jetpack_Sync_Actions::get_sync_status();
+		$args   = $this->query_args();
+		$fields = $args['fields'];
```

Comment:
Should we do an extra check whether `$args['fields']` is defined?

```suggestion
		$fields = isset( $args['fields'] ) ? $args['fields'] : array();
```